specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-04-01-preview/registries.json

```diff
@@ -507,56 +555,145 @@
       },
       "additionalProperties": false
     },
-    "Registry": {
-      "description": "Details of the Registry",
+    "PartialUserAssignedIdentity": {
       "type": "object",
-      "allOf": [
-        {
-          "$ref": "#/definitions/ResourceBase"
+      "additionalProperties": false
+    },
+    "PrivateEndpointConnectionARMResource": {
+      "description": "Private endpoint connection definition.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "This is the private endpoint connection name created on SRP\r\nFull resource id: /subscriptions/{subId}/resourceGroups/{rgName}/providers/Microsoft.MachineLearningServices/{resourceType}/{resourceName}/privateEndpointConnections/{peConnectionName}",
+          "type": "string",
+          "x-nullable": true
+        },
+        "location": {
+          "description": "Same as workspace location.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "properties": {
```

Comment:
See LintDiff Error: AvoidNestedProperties.

Consider using x-ms-client-flatten to provide a better end user experience.

Otherwise you will have nested properties:

"properties": {
        "properties": { // Redundant
                "groupIds": []
        }
}